CHANGELOG.md

```diff
@@ -84,6 +84,7 @@
 - Fix traces so that call gas costing in traces matches other clients traces [#6525](https://github.com/hyperledger/besu/pull/6525)
 
 ### Download Links
+https://github.com/hyperledger/besu/releases/tag/24.3.0
```

Comment:
We normally add the direct links and SHAs here...is there a reason not to here?

Reply:
well on the one hand I was being lazy... on the other hand it's automated in the release creation, it seemed manual to copy it in here. it's only one click away. but there's no version control on the release notes over there so I guess it makes sense to have it here as well